Universal Profiling: Add host-agent versioning information (#2939)
elastic/observability-docs: docs/en/observability/profiling-get-started.asciidoc

* For {k8s} deployments, the Helm chart version is already used to configure the same container image, unless
overwritten with the `version` parameter in the Helm values file.

> * For {stack} releases 8.8 or higher, ensure you are using `v3` host agents. For 8.7, you need to use `v2`. This is
> reflected in the instructions shown in Kibana. `v3` host agents are incompatible with 8.7 {stack} versions.

Reviewer: ```suggestion
* For {stack} version 8.8 or higher, use `v3` host agents. For version 8.7, use `v2`. `v3` host agents are incompatible with 8.7 {stack} versions.

```

A couple of small changes. Not sure we need the sentence on Kibana instructions, but if you feel it adds value, feel free to leave it in.
